components/server/src/api/server.ts

```diff
@@ -250,14 +267,16 @@ export class API {
         };
     }
 
-    private async verify(context: HandlerContext): Promise<string> {
+    private async verify(context: HandlerContext): Promise<string | undefined> {
         const cookieHeader = (context.requestHeader.get("cookie") || "") as string;
-        const claims = await this.sessionHandler.verifyJWTCookie(cookieHeader);
-        const subjectId = claims?.sub;
-        if (!subjectId) {
-            throw new ConnectError("unauthenticated", Code.Unauthenticated);
+        try {
+            const claims = await this.sessionHandler.verifyJWTCookie(cookieHeader);
```

Comment:
it cannot throw by design, try-catch and logging are superfluous, it is already logged inside

Reply:
No, please have a closer look at `AuthJWT.verify`. The signature validation is supposed to throw in case of errors. It's only not throwing if cookie isn't present at all in `verifyJWTCookie`.